component/admin/views/webservices/view.html.php

```diff
@@ -81,7 +81,19 @@ public function display($tpl = null)
 		$this->return = base64_encode('index.php?option=com_webservices&view=webservices');
 
 		// Check if option is enabled
-		if (JBootstrap::getConfig('enable_webservices', 0) == 0)
+		try
+		{
+			$container = (new Joomla\DI\Container)
```

Comment:
There's a few 5.4 dependencies at the moment (e.g. the trait herman mentioned). If we're going for 5.3 support we can just do another PR outside of this